src/dbus_api/pool/pool_2_0/methods.rs

```diff
@@ -285,11 +285,11 @@ pub fn rename_pool(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
         .expect("implicit argument must be in tree");
     let pool_uuid = get_data!(pool_path; default_return; return_message).uuid;
 
-    let msg = match dbus_context
-        .engine
-        .borrow_mut()
-        .rename_pool(pool_uuid, new_name)
-    {
+    let mut engine = dbus_context.engine.borrow_mut();
+
+    let result = pre_pool_op! {"preparing to rename pool with name {}", new_name; "successfully renamed pool with name {}, uuid {}", new_name; engine.rename_pool(pool_uuid, new_name)};
```

Comment:
Please use the `macro!()` form as opposed to the `macro!{}` form. While these are identical from a code compilation perspective, the convention is usually to use the `macro!()` form for something that acts vaguely like a function. I'd argue that this acts more like a function than another construct as it simply adds some logging around the call we pass in.

Reply:
Resolved.